test/browser/2-extend-deadline.test.ts

```diff
@@ -72,6 +72,15 @@ describe('Extend deadline', () => {
       const deadline = await extendDeadlinePage.getElementText('#extend-message');
       expect(deadline).to.contain(`${moment().utc().add(14, 'day').format(CONST.DATE_FORMAT)}`);
     });
+
+    it('shows the contact tribunal details', async() => {
+      await page.goto(`${testUrl}${Paths.taskList}`);
```

Comment:
Can this be `await taskListPage.visitPage()`?

Reply:
done